Assignment1/test/test_distance_formula.py

```diff
@@ -1,11 +1,92 @@
 import pytest
+from pytest import approx
 
 
 @pytest.fixture
 def df():
-    from Assignment1.distance_formula import DistanceFormula
-    return DistanceFormula()
+    from Assignment1.distance_formula import DistanceFormulaCalculator
+    return DistanceFormulaCalculator()
```

Comment:
Should remove the parenthesis here. We want to test against the class itself, not an instance of it.